sdk/nodejs/appservice/zMixins_timer.ts

```diff
@@ -203,13 +203,6 @@ export type TimerSubscriptionArgs = util.Overwrite<mod.CallbackFunctionAppArgs<T
      */
     resourceGroupName?: pulumi.Input<string>;
 
-    /**
-     * Specifies the supported Azure location where the resource exists. Changing this forces a new
-     * resource to be created.  If not supplied, the location of the provided ResourceGroup will be
-     * used.
-     */
-    location?: pulumi.Input<string>;
```

Comment:
out of curiosity, why did all the 'location' stuff get removed?

Reply:
It used to be an override which would change the type to allow undefined. Meanwhile, location got to allow undefined in the base class, so overrides can be removed.

Reply:
got it, thanks!